### PR TITLE
Fix: Readme and Contributing Guide

### PR DESCRIPTION
**Pull request by jessica-tw**
Signed-off-by: jessica-tw <jessica.olinda@zup.com.br>

**- What I did**
- Fixed Readme file.
- Added new Contributing Guide.

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -39,47 +36,97 @@ Check out the requisites before contributing to Horusec:
...
+- **BUGFIX:** PR refers to corrections for the next release.
+- **HOTFIX:** PR refers to corrections where you will need a cherry-pick and the update of the minor version. 
+- **CHORE:** PR refers to changes for the next release, but it was only maintenance without an activity impact.  
+
+**Example:** **[start][bugfix]: Fix bug when Horusec haven't read the new flag of authorization**
+
+ 
+2. Answer the questions about what you did, how to verify it and a short description for the changelog, see an example below:
+
+<p align="center" margin="20 0"><img src="assets/pr-template.PNG" alt="architecture" width="100%" style="max-width:100%;"/></p>
```
Comment (matheusalcantarazup) [inline CONTRIBUTING.md:89]: This image was not added on commit.

**Review by matheusalcantarazup — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -3,31 +3,28 @@
...
-4. [**Community**](#Community)
-
-## **Getting started**
-
-### **Before you contribute**
+### 1. [**Before you contribute**](#before-you-contribute)
+> #### 1.1.  [**Code of Conduct**](#code-of-conduct)
+> #### 1.2. [**Legal**](#legal)
+### 2. [**Prerequisites**](#prerequisites)
+> #### 2.1.   [**Developer Certificate of Origin**](#developer-certificate-of-origin)
```
Comment (nathanmartinszup) [inline CONTRIBUTING.md:10]: Redirect not working

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -3,31 +3,28 @@
...
+> #### 2.2.  [**Code Review**](#code-review)
+> #### 2.3. [**Pull Requests**](#pull-requests)    
+### 3. [**How to contribute?**](#how-to-contribute?)
```
Comment (nathanmartinszup) [inline CONTRIBUTING.md:13]: Redirect not working

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -3,31 +3,28 @@
...
+ > #### 3.1. [**Prepare your development environment**](#prepare-your-development-environment)
+> #### 3.2.  [**First contribution**](#first-contribution)
+> #### 3.3. [**Add new feature, bug fixing or improvement**](#add-new-feature-bugfixing-or-improvement)
```
Comment (nathanmartinszup) [inline CONTRIBUTING.md:16]: Redirect not working

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -3,31 +3,28 @@
...
+> #### 3.4.  [**Pull Request's approval**](#pull-request-approval)
```
Comment (nathanmartinszup) [inline CONTRIBUTING.md:17]: Redirect not working

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -3,31 +3,28 @@
...
+> #### 3.5.   [**After your pull request's approval**](#after-your-pull-request-approval)
```
Comment (nathanmartinszup) [inline CONTRIBUTING.md:18]: Redirect not working

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -39,47 +36,97 @@ Check out the requisites before contributing to Horusec:
...
+make coverage
+
+go tool cover -html=coverage.out # Open coverage status in your browser
+``` 
+
+#### **Repositories**
 Horusec has other repositories and you can check the README for each one of them: 
 
-- [**Admin**](https://github.com/ZupIT/horusec-admin)
 - [**Charts**](https://github.com/ZupIT/charlescd/tree/main/circle-matcher)
```
Comment (nathanmartinszup) [inline CONTRIBUTING.md:125]: Redirecting to a charles cd repostitory. I think this repository it's unnecessary. @wiliansilvazup What do you think?

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -39,47 +36,97 @@ Check out the requisites before contributing to Horusec:
...
 - [**Devkit**](https://github.com/ZupIT/horusec-devkit)
+- [**Engine**](https://github.com/ZupIT/horusec-engine)
 - [**Jenkins**](https://github.com/ZupIT/horusec-jenkins-sharedlib)
+- [**Operator**](https://github.com/ZupIT/horusec-operator)
 - [**Platform**](https://github.com/ZupIT/horusec-platform)
 - [**VSCode plugin**](https://github.com/ZupIT/horusec-vscode-plugin)
 - [**Kotlin**](https://github.com/ZupIT/horusec-tree-sitter-kotlin)
```
Comment (nathanmartinszup) [inline CONTRIBUTING.md:132]: Empty repository, necessary? @wiliansilvazup

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -115,12 +163,12 @@ Your pull request is approved when:
...
 
 ### **After your pull request's approval**
-- If it is a bugfix, the team will perform the changes and there will be a new release.
+- If it is a bug fix, the team will perform the changes and there will be a new release.
 - If it is a feature, it will be in the next release. 
 
 ## **Community**
 
-- Do you have any question about Horusec in our [**mailing list**](horusec@zup.com.br) 
+- Do you have any question about Horusec? Send to our [**mailing list**](horusec@zup.com.br).
```
Comment (nathanmartinszup) [inline CONTRIBUTING.md:171]: Mail list returning not found

**Comment by nathanmartinszup** [comment]
Rebase branch with main

**Review by nathanmartinszup — APPROVED** [review]

**Pull request merged**